### PR TITLE
bemhtml: fix elemMods behaviour

### PR DESCRIPTION
**Pull request by zxqfox**
Fixes: #111

### CODE REVIEW

File: test/bemhtml/runtime-test.js [hunk 1]
```diff
@@ -380,6 +391,20 @@ describe('BEMHTML compiler/Runtime', function() {
         }
       }, '<div class="b1 b1__e1"></div>');
     });
+
+    it('should support changing elemMods in runtime', function() {
+      test(function() {
+        block('b1').elem('e1').def()(function() {
+          this.elemMods.a = 'b';
+          console.log(this.elemMods);
```
Comment (tadatuta) [inline test/bemhtml/runtime-test.js]: ~~console.log(this.elemMods);~~

File: test/fixtures.js [hunk 2]
```diff
@@ -39,8 +54,17 @@ module.exports = function(engine) {
     }, regexp);
   }
 
+  function compile(fn, options) {
```
Comment (veged) [inline test/fixtures.js]: is it relevant to issue? maybe better put it in separate commit/pr

**Comment by miripiruni** [comment]
:+1:

File: test/fixtures.js (hunk 2 above)
Comment (zxqfox) [inline test/fixtures.js]: Yeah, better to split it.

**Comment by zxqfox** [comment]
Closed in favor of #157
